src/08_comprehensions.py

```diff
@@ -12,13 +12,19 @@
 
 y = []
 
+for n in range(5):
+    y.append(n+1)
+
```

Comment:
look up populating list comprehensions with for loops on the same line. You shouldn't need to use two lines!